botocore/serialize.py

```diff
@@ -431,6 +433,218 @@ def _serialize_type_blob(self, serialized, value, shape, key):
         serialized[key] = self._get_base64(value)
 
 
+class CBORSerializer(Serializer):
```

Comment:
The CBOR RFC defines the major types pretty well here: https://www.rfc-editor.org/rfc/rfc8949.html#name-major-types

In most methods were starting them with:

```python
major_type = # some number 1-7

...

... _get_initial_byte(major_type, ...)
```

Can we make these descriptive class-level constants for the `CBORSerializer` instead?